readme.md

```diff
@@ -153,91 +150,99 @@ button.on( "click", handleClickEvent );
 ```
 
 <details>
-<summary>Vanilla JS implementation</summary>
 
-```
-document.querySelector("button").addEventListener("click", handleClickEvent)
+  <summary><strong>Vanilla Javascript implementation</strong></summary>
+
+  ```js
+  var button = document.querySelector("button")
+  button.addEventListener("click", handleClickEvent)
```

Comment:
I wonder if it's worth showing them putting in an anonymous function... not sure myself